src/sentry/static/sentry/app/components/groupTombstones.jsx

```diff
@@ -0,0 +1,115 @@
+        });
+      },
+      error: () => {
+        AlertActions.addAlert({
+          message: t('We were unable to discard this group'),
+          type: 'error'
+        });
+      }
+    });
+    this.fetchData();
+  },
+
+  renderEmpty() {
+    return <div className="box empty">{t('None')}</div>;
+  },
+
+  render() {
+    if (this.props.tombstoneError) return <LoadingError />;
+
+    let {tombstones} = this.props;
+    return (
+      <div>
+        <div className="row" style={{paddingTop: 10}}>
+          <div className="col-sm-8">
+            <h5>{t('Discarded Groups')}</h5>
+            {tombstones.length
+              ? <ul className="group-list">
+                  {tombstones.map(data => {
+                    return (
+                      <li
+                        key={data.id}
+                        className={`group row level-${data.level} type-${data.type}`}>
+                        <div className="col-md-7 col-xs-8 event-details">
+                          <div>
+                            <h3 className="truncate">
+                              <span className="error-level truncate" title={data.level}>
+                                {data.level}
+                              </span>
+                              <span>
+                                <span style={{marginRight: 10}}>{data.type}</span>
+                                <em>{data.culprit}</em><br />
+                              </span>
+                            </h3>
+                            <div className="event-message truncate">
+                              <span className="message">{data.message}</span>
+                            </div>
+                            <div className="event-extra">
+
+                              <LinkWithConfirmation
+                                title={t('Undiscard')}
+                                className="btn btn-warning btn-xs"
+                                message={t(
+                                  'Undiscarding this group means that ' +
+                                    'incoming events that match this will no longer be discarded. ' +
+                                    'New incoming events will count toward your event quota ' +
+                                    'and will display on your issues dashboard. ' +
+                                    'Are you sure you wish to continue?'
+                                )}
+                                onConfirm={() => {
+                                  this.undiscard(data.id);
+                                }}>
+                                <span>{t('Undiscard')}</span>
+                              </LinkWithConfirmation>
+                              {data.actor &&
+                                <span
+                                  style={{marginLeft: 10}}
+                                  className="event-message actor">
+                                  {t(`discarded by ${data.actor.name}`)}
```

Comment:
Careful – there's a specific way to declare translatable strings that take arguments. Take a look at examples of `tct` in our codebase. 

Looks like this:

```javascript
import {tct} from '../locale';

// ...

return tct('[actor] marked this issue as resolved in the upcoming release.', {
  actor: actor
});
```